### PR TITLE
Michelle scott

### PR DESCRIPTION
**Pull request by scottmm374**

### CODE REVIEW

File: less/index.less [hunk 1]
```diff
@@ -48,12 +48,253 @@ table {
...
+}
+
+.list1 {
+	.gradient;
+}
+
+
+
+
+.container {
```
Comment (ari7946) [inline less/index.less]: The lack of indentation on the selectors is not common. However, this is preference and it shouldn't affect functionality. That said, the code looks clean.

File: index.html [hunk 2]
```diff
@@ -1,19 +1,116 @@
...
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Me</a>
+        <a href="https://scottmm374.github.io/" target="blank">Portfolio</a>
+        <a href="mailto:scottmm374@gmail.com">Contact</a>
+      </nav>
+    </section>
+
+    <header>
+      <div class="left-content">
```
Comment (ari7946) [inline index.html]: would changing "left-content" and "right-content" to a name that's a bit more descriptive be a good idea?

**Comment by ari7946** [comment]
Amazing work, Michelle! The site looks beautiful. The code is clean. I like the number of descriptive commit messages! Great job!

File: index.html (hunk 2 above)
Comment (scottmm374) [reply index.html]: Yes, I can try to do that :)

File: less/index.less (hunk 1 above)
Comment (scottmm374) [reply less/index.less]: I hadn't noticed I had it all the way up against the starting point, :) Thanks.

**Pull request merged**